src/blocking/prelude.rs

```diff
@@ -0,0 +1,2 @@
+#[doc(no_inline)]
+pub use crate::{blocking::*, prelude::*};
```

Comment:
An alternative to this file would be to just have the top-level prelude export `blocking::*` with conditional compilation.

Reply:
Went with that approach